data/json/items/comestibles.json

```diff
@@ -15196,6 +15196,95 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "milk_curdled",
+        "name" : "curdled milk",
+        "weight" : 236,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 672,
+        "stim" : 0,
+        "container" : "waterskin",
+        "to_hit" : 0,
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 8,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 4,
+        "description" : "Milk that has been curdled with vinegar and rennet.  It still needs to be salted and drained of whey.",
+        "price" : 10,
+        "material" : "milk",
+        "tool" : "null",
+        "volume" : 6,
+        "cutting" : 0,
+        "phase" : "liquid",
+        "charges" : 1,
+        "stack_size" : 6,
+        "bashing" : 0,
+        "fun" : -12
+    },
+    {
+        "type" : "COMESTIBLE",
+        "id" : "brew_milk_curdled",
+        "name" : "curdling milk",
+        "weight" : 236,
+        "color" : "white",
+        "addiction_type" : "none",
+        "spoils_in" : 672,
+        "stim" : 0,
+        "container" : "waterskin",
+        "to_hit" : 0,
+        "comestible_type" : "DRINK",
+        "symbol" : "~",
+        "quench" : 8,
+        "healthy" : 0,
+        "addiction_potential" : 0,
+        "nutrition" : 4,
+        "description" : "Milk with vinegar and natural rennet added.  Used for making cheese.",
```

Comment:
It already says "Once set in a vat, this will ferment in around 3 days". However, I have added another hint that this item should be placet in a vat.